utils/gitlog2changelog.py

```diff
@@ -15,7 +16,20 @@
     rev_range = "%s..HEAD" % base
 
 # Execute git log with the desired command line options.
-fin = os.popen("git log --summary --stat --no-merges --date=short %s" % rev_range, "r")
+# uses subproccess instead, more secure.
+command =  ["git", "log", "--summary", "--stat", "--no-merges", "--date=short", rev_range]
+
+fin = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Consider subprocess.run or other functions from the subprocess package. Popen is the most basic option and the functions (added later in the subprocess package history) are usually more convenient.

Reply:
I was able to get it running with subprocess.Popen, however if a different package would be more beneficial to grass, let me know

Reply:
Now it is running fine with subprocess.Popen, so let's leave it as is.